### PR TITLE
Fixes #1243: Simplify Signup Dialog

### LINKED ISSUE #1243: Simplify Login and Sign up boxes
Please simplify the login and signup boxes. Take out icons and check the approach of Facebook to keep it more simple.

screenshot from 2018-05-25 21-39-32

screenshot from 2018-05-25 21-42-52

### PR DESCRIPTION
**Pull request by anshumanv**
Fixes #1243 

Changes: Simplified Sign up dialog.

Demo Link: https://pr-1303-fossasia-susi-web-chat.surge.sh

Screenshots for the change: 
image

### CODE REVIEW

**Comment by duttaditya18** [comment]
@anshumanv Preview Link?

**Comment by anshumanv** [comment]
@duttaditya18 https://pr-1303-fossasia-susi-web-chat.surge.sh/

Even if the PR number is not filled out you can check for the PR number and get a link accordingly.

File: src/components/Auth/SignUp/SignUp.css [hunk 1]
```diff
@@ -20,4 +20,15 @@
...
+.login-links{
+	color: #365899;
+	cursor: pointer;
+	font-size: 12px;
+}
+
+.login-links:hover{
+	color: #365899;
+	text-decoration: underline;
 }
```
Comment (Akshat-Jain) [inline src/components/Auth/SignUp/SignUp.css]: Please add EOF here. Thanks :-)

Comment (Akshat-Jain) [reply src/components/Auth/SignUp/SignUp.css]: Also, please use proper indentation wherever required. Thanks :-)

**Pull request merged**